monitoring/src/main/java/org/polypheny/db/monitoring/events/DdlEvent.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2019-2022 The Polypheny Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.polypheny.db.monitoring.events;
+
+import java.util.Arrays;
+import java.util.List;
+import org.polypheny.db.monitoring.events.analyzer.DdlEventAnalyzer;
+import org.polypheny.db.monitoring.events.metrics.DdlDataPoint;
+
+public class DdlEvent extends StatementEvent {
+
+    private String eventType = "DDL EVENT";
```

Comment:
Can this be static final EVENT_TYPE? However, it is never used

Reply:
Deleted the eventType.